Sign in page to serve

grisanu opened the pull request:
Button not yet centered

erinkav committed:
ES6 Modifications to style guide

erinkav committed:
ES6 Modifications to style guide

erinkav committed:
Formatting edits to style guide

erinkav committed:
Formatting edits to style guide

erinkav committed:
Changes to margins on Style Guide

erinkav committed:
Changes to margins on Style Guide

erinkav committed:
(feat) Initial webpack configuration and test files

lessenadam committed:
(feat) Initialize server and db

lessenadam committed:
(doc) Update the readme

grisanu committed:
Start making client-side pages

-Modified package.json to include react, react-dom and react-bootstrap
-Created a style.css and index.html

grisanu committed:
Start making client-side pages

-Modified package.json to include react, react-dom and react-bootstrap
-Created a style.css and index.html

grisanu committed:
Html now load React scripts as libraries

grisanu committed:
Html now load React scripts as libraries

grisanu committed:
Make a button

-Add gitignore file
-index.html now loads lib from CDN because then now automically places lib on window object
-Made a button appear on page with react-bootstrap

grisanu committed:
Make a button

-Add gitignore file
-index.html now loads lib from CDN because then now automically places lib on window object
-Made a button appear on page with react-bootstrap

erinkav committed:
Webpack and webpack dev server running

lessenadam committed:
(feat) Implement logger for server requests and hooked up DB to send through its data

erinkav committed:
Deleted test files

grisanu committed:
Trying to make button center on page

grisanu committed:
Trying to make button center on page

lessenadam committed:
Merge pull request #42 from erinkav/webpack-feature-branch-2

(feat) Functioning Webpack config with dev server

erinkav committed:
Merge pull request #41 from lessenadam/feat/serverinit

Feat/serverinit

grisanu committed:
(cleanup) Changed name to resolve naming conflict

grisanu committed:
(cleanup) Changed name to resolve naming conflict

lessenadam commented on public/index.html:
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE>
```
change index.html to something else to avoid temp conflicts

grisanu committed:
(test) Adjust div to fill page

grisanu committed:
(test) Adjust div to fill page

grisanu committed:
(cleanup) Resolve merge conflicts
Merge branch 'sign-in-page' of https://github.com/mattgrisanu/fotos into sign-in-page
